fastenhealth/fasten-onprem
# Add PostgreSQL as DB Option

Testing steps:
- `go mod vendor`
- update `config.yaml` to have `postgres` as database type
- from `fasten-onprem/backend/pkg/models/database` run `go run generate.go` and verify that all the database models have swapped out `datetime` for `timestamptz`
